styles/namemc/catppuccin.user.css

```diff
@@ -0,0 +1,582 @@
+    --bs-success-bg-subtle: @crust;
+    --bs-warning-bg-subtle: @surface0;
+    --bs-danger-bg-subtle: @base;
+    --bs-secondary-border-subtle: @surface1;
+    --bs-success-border-subtle: @accent-color;
+    --bs-info-border-subtle: @sky;
+    --bs-warning-border-subtle: @peach;
+    --bs-danger-border-subtle: @red;
+    --bs-link-color: @accent-color;
+    --bs-link-color-rgb: @accent-color;
+    --bs-code-color: @accent-color;
+    --bs-highlight-bg: @base;
+    --bs-form-valid-color: @green;
+    --bs-focus-ring-color: @accent-color;
+    --bs-form-valid-border-color: @green;
+    --bs-form-invalid-color: @red;
+    --bs-form-invalid-border-color: @red;
+
+    #bsButton(@col, @bgCol) {
+      --bs-btn-color: @col;
+      --bs-btn-hover-color: @col;
+      --bs-btn-active-color: @col;
+      --bs-btn-disabled-color: @col;
+
+      --bs-btn-bg: @bgCol;
+      --bs-btn-hover-bg: @bgCol;
+      --bs-btn-active-bg: @bgCol;
+      --bs-btn-disabled-bg: @bgCol;
+      --bs-btn-border-color: @bgCol;
+      --bs-btn-hover-border-color: @bgCol;
+      --bs-btn-active-border-color: @bgCol;
+      --bs-btn-disabled-border-color: @bgCol;
+    }
+    #bsButtonOutline(@col, @hover) {
+      --bs-btn-color: @col;
+      --bs-btn-hover-bg: @col;
+      --bs-btn-active-bg: @col;
+      --bs-btn-border-color: @col;
+      --bs-btn-disabled-color: @col;
+      --bs-btn-focus-shadow-rgb: @col;
+      --bs-btn-hover-border-color: @col;
+      --bs-btn-active-border-color: @col;
+      --bs-btn-disabled-border-color: @col;
+
+      --bs-btn-hover-color: @text;
+      --bs-btn-active-color: @text;
+    }
+
+    .btn-primary {
+      #bsButton(@crust, @accent-color);
+    }
+    .btn-secondary {
+      #bsButton(@text, @overlay0);
+    }
+    .btn-success {
+      #bsButton(@crust, @green);
+    }
+    .btn-info {
+      #bsButton(@crust, @sapphire);
+    }
+    .btn-warning {
+      #bsButton(@crust, @yellow);
+    }
+    .btn-danger {
+      #bsButton(@crust, @red);
+    }
+    .btn-light {
+      #bsButton(@crust, @overlay2);
+    }
+    .btn-dark {
+      #bsButton(@text, @surface0);
+    }
+
+    .btn-outline-primary {
+      #bsButtonOutline(@accent-color, @text);
+    }
+    .btn-outline-secondary {
+      #bsButtonOutline(@overlay0, @text);
+    }
+    .btn-outline-success {
+      #bsButtonOutline(@green, @text);
+    }
+    .btn-outline-info {
+      #bsButtonOutline(@sapphire, @crust);
+    }
+    .btn-outline-warning {
+      #bsButtonOutline(@yellow, @crust);
+    }
+    .btn-outline-danger {
+      #bsButtonOutline(@red, @text);
+    }
+    .btn-outline-light {
+      #bsButtonOutline(@text, @crust);
+    }
+    .btn-outline-dark {
+      #bsButtonOutline(@surface0, @text);
+    }
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
```

Comment:
This should be the first section of stuff, right beneath the variables (line 54).